docs/source/guides/importpilotmodel.rst

```diff
@@ -0,0 +1,236 @@
+.. _importpilotmodel:
+
+Pilot Model Importing
+=====
+
+Tools:
+^^^^^
+
+.. _Blender: https://www.blender.org/
+.. _Blender Source Tools: http://steamreview.org/BlenderSourceTools/
+.. _Titanfall VPK Tools: https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools
+.. _Crowbar: https://steamcommunity.com/groups/CrowbarTool
+.. _Legion+: https://github.com/r-ex/LegionPlus
+.. _RePak: https://github.com/r-ex/RePak
+.. _mdlshit: https://github.com/headassbtw/mdlshit-binaries
+.. _Text Editors: https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing
+
+Download each tool. 
+
+`Text Editors`_ - Just a text editor
+
+`Titanfall VPK Tools`_ - VPK viewer, extractor, and repacker.
+   "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
+
+`Crowbar`_ - Decompile, recompile, and a ton of other model and source asset stuff
+
+`Blender`_ - 2D/3D Workspace
+
+`Blender Source Tools`_ - Source Tools plugin for Blender
+
+`Legion+`_  - Texture, audio, shaderset, and material exporter
+   "Normal" Legion doesn't work
+
+`RePak`_ - Create Apex and Titanfall 2 .rpak for textures
+
+`mdlshit`_ - Magically convert .mdl to "titanfall 2" .mdl 
```

Comment:
Might look better formatted as bullet point list, i.e.


> - [Text Editors](https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing) - Just a text editor
> - [Titanfall VPK Tools](https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools) - VPK viewer, extractor, and repacker.
>    "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
> - [Crowbar](https://steamcommunity.com/groups/CrowbarTool) - Decompile, recompile, and a ton of other model 
and source asset stuff
> 
> ...